### PR TITLE
[SD-3399] - Text archive import command registered

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: server/apps/aap/import_text_archive/commands.py [hunk 1]
```diff
@@ -152,7 +153,7 @@ def _process_bunch(self, x):
             item['firstcreated'] = utc.normalize(value) if value.tzinfo else value
             item['versioncreated'] = item['firstcreated']
 
-            item['unique_id'] = doc.find('dcdossier').get('unique')
+            generate_unique_id_and_name(item)
             item['ingest_id'] = id
```
Comment (plamut) [inline server/apps/aap/import_text_archive/commands.py]: Just checking - is the `ingest_id` attribute supposed to be a function?

**Comment by plamut** [comment]
:+1:

File: server/apps/aap/import_text_archive/commands.py (hunk 1 above)
Comment (akintolga) [inline server/apps/aap/import_text_archive/commands.py]: I think it is an attribute.

Comment (plamut) [inline server/apps/aap/import_text_archive/commands.py]: Sorry, should have written more clearly. :) Is the _value_ of this attribute meant to be a function?

**Comment by sivakuna-aap** [comment]
:+1: Can we replace strings with constants in `server/apps/aap/import_text_archive/commands.py`?

**Comment by marwoodandrew** [comment]
Wouldn't bother making it to pretty, should only ever be used once!

:+1:

**Pull request merged**